lib/go-tc/alerts.go

```diff
@@ -64,8 +65,9 @@ func GetHandleErrorsFunc(w http.ResponseWriter, r *http.Request) func(status int
 			fmt.Fprintf(w, http.StatusText(http.StatusInternalServerError))
 			return
 		}
-		w.Header().Set("Content-Type", "application/json")
-		w.WriteHeader(status)
+
+		w.Header().Set(ContentType, ApplicationJson)
+		w.Header().Set("status", strconv.Itoa(status))
```

Comment:
Is this going to make errors be returned as `200` by the Wrapper? I think we need to pass the code up (maybe in the Context?), to be finally set by the Wrapper, after gzipping. Unless there's a way to set the HTTP Response Code before writing all the headers

Reply:
you are not mistaken, I looked at the wrong call in my developer tools when I checked the status code.